macosx/AboutWindowController.swift

```diff
@@ -0,0 +1,71 @@
+// This file Copyright © 2007-2023 Transmission authors and contributors.
+// It may be used under the MIT (SPDX: MIT) license.
+// License text can be found in the licenses/ folder.
+
+import AppKit
+
+class AboutWindowController: NSWindowController {
+    private static var fAboutWindowInstance: AboutWindowController?
+    @objc static var aboutController: AboutWindowController {
+        if let fAboutWindowInstance = fAboutWindowInstance {
+            return fAboutWindowInstance
+        }
+        let aboutWindowInstance = AboutWindowController(windowNibName: "AboutWindow")
+        self.fAboutWindowInstance = aboutWindowInstance
+        return aboutWindowInstance
+    }
+
+    @IBOutlet private weak var fTextView: NSTextView!
```

Comment:
Swift migration might be a good time to drop all `f` prefixes from fields, no one really does it in Swift code

Reply:
I don't want to touch to any XIB file in this PR. All XIB changes can be done in a subsequent PR.